src/bin/tikv-ctl.rs

```diff
@@ -750,6 +758,14 @@ fn main() {
                 .takes_value(true)
                 .help("convert escaped key to hex key"),
         )
+        .arg(
+            Arg::with_name("pd")
```

Comment:
Seems other commands tend to keep the "pd" argument in the subcommand, I think we should do that too.
If you think we should move the "pd" argument to the upper level, you can discuss with others and maybe do that in another PR.

Reply:
Yes, I tried that. Other commands put `pd` in subcommand  because they have one of the `db` or `host` argument. If we put `pd` in subcommand we must have one of the `db` or `host`, since they both have `.required(true)` field.